button/button.css

```diff
@@ -0,0 +1,57 @@
+.btn-primary {
+	color: var(--color-white);
+	background-color: var(--color-blue-800);
+}
+
+.btn-primary:hover {
+	background-color: var(--color-blue-600);
+}
+
+.btn-secondary {
+	color: var(--color-white);
+	background-color: var(--color-red-600);
+}
+
+.btn-secondary:hover {
+	background-color: var(--color-red-800);
+}
+
+.btn-ternary {
+	background-color: var(--color-gray-200);
+}
+
+.btn-ternary:hover {
+	background-color: var(--color-gray-400);
+}
+
+.btn-outlined {
+	color: var(--color-blue-800);
+	border: 1px solid var(--color-blue-800);
+}
+
+.btn-outlined:hover {
+	color: var(--color-white);
+	background-color: var(--color-blue-800);
+}
+

```

Comment:
Same suggestion as above, plus we know its a color.. you can emit the work color-white... just white is also ok...

you can mention color with like primary color or font-color something like this variables

Reply:
Changes done.